ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -71,6 +73,23 @@ message Commands {
   // Must not be set at the same time as min_ledger_time_abs.
   // Optional
   google.protobuf.Duration min_ledger_time_rel = 11;
+
+  // Set of parties on whose behalf the command should be executed. It is up to the server to verify that the
+  // authorisation can be granted and that the connection has been authenticated for the given set of parties.
+  // This field supersedes the ``party`` field. The effective set of parties on whose behalf the command
+  // should be executed is the union of all parties listed in ``party`` and ``act_as``. This set must not be
+  // empty.
+  // Each element must be a valid PartyIdString (as described in ``value.proto``).
+  // Optional
+  repeated string act_as = 12;
+
+  // Set of parties on whose behalf (in addition to all parties listed in ``act_as``) contracts can be retrieved.
+  // This affects DAML operations such as ``fetch``, ``fetchByKey``, ``lookupByKey``, ``exercise``, and ``exerciseByKey``.
+  // Note: A participant node of a DAML network can host multiple parties. Each contract present on the participant
+  // node is only visible to a subset of these parties. A command can only use contracts that are visible to at least
+  // one of the parties in ``act_as`` or ``read_as``.
```

Comment:
Consider adding a comment that clarifies that authorization rules for fetches are still checked.